src/ng-add/index.ts

```diff
@@ -1,9 +1,44 @@
 import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+
+const svgToTsVersion = '^5.7.0';
```

Comment:
TODO: Likely if there are breaking schema changes in `svg-to-ts` this version would matter. Right now they are aligned.